doc/Manuals/ug-syspar_x-road_v6_system_parameters.md

```diff
@@ -484,14 +485,20 @@ For instructions on how to change the parameter values, see section [Changing th
-| rate-limit-requests-per-second | 20                | Controls how many requests from an IP address are allowed per second. To disable this feature, set this value to -1.                       |
-| rate-limit-requests-per-minute | 600               | Controls how many requests from an IP address are allowed per minute. To disable this feature, set this value to -1.                       |
-| rate-limit-cache-size          | 10000             | Controls how many IP addresses can be remembered in the rate-limit cache Tradeoff between memory usage and protection from a large attack. |
-| cache-default-ttl              | 60                | Configures default cache expiration in seconds. Cache is hit during authentication requests. Setting the value to -1 disables the cache.   |
-| cache-api-key-ttl              | 60                | Configures Api Key cache expiration in seconds. Can be used by various api services. Setting the value to -1 disables the cache.           |
+| **Name**                          | **Default value**                                   | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+|-----------------------------------|-----------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| allowed-hostnames                 |                                                     | Determines which hostnames are allowed to be used in "Host" header of the HTTP requests. Any hostname is allowed when left unspecified.                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
If multiple hostnames are specified, how different hostnames should be separated from one another? Is it a comma separated list or something else?